core/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 project(tangram-core)
 
 # Build core library dependencies.
+if (WIN32)
+  find_package(yaml-cpp CONFIG REQUIRED)
+  find_package(ZLIB REQUIRED)
```

Comment:
I think that zlib thingy can be moved outside an `if` - most non-windows OSes have zlib in its system libs, find_package should be able to find them and ZLIB::ZLIB should be resolved correctly.